src/main/java/de/zalando/aruha/nakadi/config/JsonConfig.java

```diff
@@ -0,0 +1,35 @@
+package de.zalando.aruha.nakadi.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsonorg.JSONObjectDeserializer;
+import com.fasterxml.jackson.datatype.jsonorg.JSONObjectSerializer;
+import org.json.JSONObject;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.zalando.problem.ProblemModule;
+
+@Configuration
+public class JsonConfig {
+
+    @Bean
+    @Primary
+    public ObjectMapper jacksonObjectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper().setPropertyNamingStrategy(
+                PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+
+        SimpleModule jsonObjectModule = new SimpleModule();
```

Comment:
I'm not sure anymore, but I think I added this module when schema used to be a `JSONObject`. Maybe we don't need this anymore. I see that this is not in the scope of this task, but I just realised it now.